.github/workflows/pull-request-automation.yml

```diff
@@ -0,0 +1,15 @@
+on: pull_request
+name: Pull request automation
+
+jobs:
+  pull-request-automation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - run: npm install
```

Comment:
This is necessary so that our action has `@actions/core` and `@actions/github`, but it makes the workflow very very slow 😢

Two alternatives I can think of:

1. Host the action in a different repository and publish the build files. This is [what GitHub recommends](https://github.com/actions/toolkit/blob/master/docs/javascript-action.md). It sucks that we can't use all of the nice infrastructure (e.g. linting, tests) that we already have, though.
2. Change this line so that it only installs the dependencies we know we need: `npm install @actions/core @actions/github`. This would be difficult to maintain, though.

Reply:
Using `npm ci` instead of `npm install` will cut a bit of time off this (20 seconds testing locally).

Reply:
can it install only the's package packages dependencies by cd'ing into the folder running into the command? e.g. `cd packages/pull-request-automation && npm install `.

There'd be a `node_modules` in the local folder, which is a bit different, but I think it's ok as it'll only exist in the context of this job where nobody will be able to see the file system.

Reply:
> can it install only the package dependencies by cd'ing into the folder running into the command? e.g. cd packages/pull-request-automation && npm install .

Did this in 3c22f92. It's not elegant but let's see how it goes. Thanks for the suggestion @talldan! ❤️